yarn.lock

```diff
@@ -1,4 +1,2 @@
 # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we should remove this file!

Reply:
I'm having trouble undoing this change because prettier keeps changing it back, how can I remove the file from this pr?